src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -201,13 +200,6 @@ const useCoreChatSystemHook = () => {
       websocketData?.originalMembers?.forEach(async (member) => {
         const userMember = UserSchema.fromMemberWebsocketObject(member, websocketData?.channel?.id);
         await userMember.saveOrUpdateIfExists(localDb);
-
-        const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
-          websocketData?.channel?.id,
-          websocketData?.message?.id,
-          member
-        );
-        await memberSchema.save(localDb);
       });
     } catch (e) {
       console.log('error on memberSchema');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The creation and saving of `memberSchema` objects have been removed. This might affect the functionality if these objects were used elsewhere. Make sure that this change doesn't break any dependencies on `memberSchema`.
```diff
- const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
-   websocketData?.channel?.id,
-   websocketData?.message?.id,
-   member
- );
- await memberSchema.save(localDb);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->